src/v/s3/logger.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include "seastarx.h"
+
+#include <seastar/util/log.hh>
+
+namespace s3 {
+extern ss::logger s3_log;
```

Comment:
i _think_ you can use `inline ss::logger s_log("s3");` in the header and remove the logger.cc file. i think we may have both patterns in the tree.